From be02b0c0a1dafc415c25cc6cb2f9caa6e7bf7896 Mon Sep 17 00:00:00 2001
From: Gilberto Garcia <gilbertodamian14@gmail.com>
Date: Sat, 21 Sep 2019 16:41:35 -0500
Subject: [PATCH] Added a fix for slots bigger than the inventory itself Maybe
 on the future I will add a config to disable this change


diff --git a/src/main/java/net/minecraft/server/Container.java b/src/main/java/net/minecraft/server/Container.java
index 7fc49579..897b68d4 100644
--- a/src/main/java/net/minecraft/server/Container.java
+++ b/src/main/java/net/minecraft/server/Container.java
@@ -106,6 +106,9 @@ public abstract class Container {
     }
 
     public Slot getSlot(int i) {
+		if (this.c.size() <= i) {
+            return (Slot) this.c.get(0);
+        }
         return (Slot) this.c.get(i);
     }
 
-- 
2.21.0.windows.1

